VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/switch-on-camera-microphone/switch-on-camera-microphone.component.ts

```diff
@@ -72,15 +73,25 @@ export class SwitchOnCameraMicrophoneComponent extends ParticipantStatusBaseDire
     }
 
     async requestMedia() {
-        this.mediaAccepted = await this.userMediaStreamService.requestAccess();
-        this.userPrompted = true;
-        if (!this.mediaAccepted) {
-            this.logger.warn(`[SwitchOnCameraMicrophone] - ${this.participantName} denied access to camera.`, {
-                conference: this.conferenceId,
-                participant: this.participantName
+        this.userMediaStreamService.currentStream$
+            .pipe(take(1))
```

Comment:
Consider first() instead of take(1)

Reply:
Yeah, I think first(0 is correct becuase take(1) won't throw any error